atomizer.config.js

```diff
@@ -7,11 +7,19 @@ module.exports = {
     },
     content: ['app/components/', 'app/assets/js', 'docs/'],
     custom: {
-        borderTopLight: '1px solid rgba(2, 128, 174, 0.3)',
+        borderTopLight: '1px solid var(--color-border-light)',
         boxColorLight: 'rgba(2, 128, 174, 0.05)',
         boxColorDarker: 'rgba(2, 128, 174, 0.2)',
```

Comment:
these two needed?

Reply:
they have been removed in a recent commit